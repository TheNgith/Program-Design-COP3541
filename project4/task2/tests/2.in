5 12 43 45 2 34 4 8 3 90 67
