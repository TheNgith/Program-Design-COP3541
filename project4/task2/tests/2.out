Enter length of array #1:Enter array elements: Enter length of array #2:Enter array elements: Output: 12 43 45 2 34 8 3 90 67